package/autoyast2.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Fri Jun 24 12:50:20 CEST 2016 - schubi@suse.de
```

Comment:
The Knights of the Chronological Order wouldn't like this. Please use today's date